workflows/iuf/operations/s3-upload/s3-upload-template.yaml

```
@@ -72,6 +72,26 @@ spec:
                 PARENT_DIR=$(echo '{{inputs.parameters.global_params}}' | jq -r '.stage_params."process-media".current_product.parent_directory')
 
                 /usr/local/bin/s3-upload "$CONTENT" "$PARENT_DIR"
+    - - name: s3-manufacture-product-catalog-data
+        template: make-product-catalog-content
+        arguments:
+          parameters:
+            - name: global_params
+              value: "{{inputs.parameters.global_params}}"
```

I think we had talked about this previously but just wanted to bring it up again - it would probably be cleaner to just pass in the s3 content from the current product manifest instead of the entire `global_params` structure, and call it something like `manifest_s3_content`. I think that would look like this:

```suggestion
            - name: manifest_s3_content
              value: "{{=jsonpath(inputs.parameters.global_params, '$.product_manifest.current_product.manifest.content.s3')}}"
```

And the name of the input parameter for `make-product-catalog-content` would have to change from `global_params` to `manifest_s3_content` (line 153).

And finally the script itself would change a little, I think it would be:

```
ARRAY=$(echo '{{inputs.parameters.manifest_s3_content}}' | jq -r '[.[] | {"bucket": .bucket, "key": .key}]')
```

(note: I haven't tested any of this so I could have made a mistake).

---

I committed your suggestion, and I'll make other additional modifications re-test it. I'll put the PR into draft state until the tests pass.

---

I think I figured out how we can make this work, if `value` uses the `toJson` function then I think you get a valid JSON structure that can be passed to the `jq` command in your script.
```
value: "{{=toJson(jsonpath(inputs.parameters.global_params, '$.product_manifest.current_product.manifest.content.s3'))}}"
```